src/auth/auth.controller.ts

```
@@ -207,4 +210,9 @@ export class AuthController {
   async getPublicRecruiterProfile(@Param('recruiterId') recruiterId: string) {
     return this.authService.getPublicRecruiterProfile(recruiterId);
   }
+
+  @Get('freelancer/:freelancerId/performance-dashboard')
+  async getFreelancerPerformanceDashboard(@Param('freelancerId') freelancerId: string): Promise<PerformanceMetricsDto> {
```

ESLint

[error] Replace `@Param('freelancerId')·freelancerId:·string` with `⏎····@Param('freelancerId')·freelancerId:·string,⏎··`

(prettier/prettier)